functions/db/community.js

```diff
@@ -14,4 +14,32 @@ const getCommunityPostDetail = async (client, communityPostId) => {
   return convertSnakeToCamel.keysToCamel(rows[0]);
 };
 
-module.exports = { getCommunityPostDetail };
+const getCommunityPosts = async (client, limit, page) => {
+  const { rows } = await client.query(
+    `
+    SELECT cp.id, u.nickname, cp.title, cp.body, cp.content_url, cp.content_title, cp.content_description, cp.thumbnail_url, cp.created_at
+    FROM community_post cp
+    JOIN "user" u on cp.user_id = u.id
+    WHERE cp.is_deleted = FALSE
+    ORDER BY cp.created_at DESC
+    LIMIT $1 OFFSET $2
+    `,
+    [limit, (page - 1) * limit],
+  );
+
+  return convertSnakeToCamel.keysToCamel(rows);
+};
+
+const getCommunityPostsCount = async (client) => {
+  const { rows } = await client.query(
+    `
+    SELECT COUNT(*) as count
```

Comment:
```suggestion
    SELECT COUNT(*)::int 
```
PostgreSQL counting 타입이 bigint 라 문자열로 인식해서 위에 number로 형변환 해준 것 같은데
이렇게 캐스팅해주면 number 로 바로 반환 될 것 같아서 한번 시도 해보실래요?
추가로 (제가 알기엔) `as count` 붙이지 않아도 count 로 반환될 것 같은데 아니었나요?

[참고](https://stackoverflow.com/questions/47843370/postgres-sequelize-raw-query-to-get-count-returns-string-value)

Reply:
COUNT(*)::int 로 number 타입의 count 반환해주네요 ! 감사합니다 🙌